Yosemite/Yosemite/Stores/MetaDataStore.swift

```diff
@@ -0,0 +1,209 @@
+import Foundation
+import Networking
+import Storage
+
+
+// MARK: - MetaDataStore
+//
+public final class MetaDataStore: Store {
```

Comment:
We have to initialize this store [here](https://github.com/woocommerce/woocommerce-ios/blob/trunk/WooCommerce/Classes/Yosemite/AuthenticatedState.swift#L100).

Reply:
Good catch. Since it's not still in use, I didn't noticed it. Done here: 60e5e0599614cd9046890fa402e84d0cc1e3f2f2